<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CleaningScreen</class>
 <widget class="QWidget" name="CleaningScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="CleanScreenLayout">
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="LowerCleaningButtonsLayout">
     <item>
      <widget class="QPushButton" name="PauseResumeCleanButton">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pause/Resume</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AbortCleanButton">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Abort Cleaning</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ReturnToMenuButton">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Main Menu</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="CleaningStatusLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="CurrentCleanTaskLabel">
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>What am I doing?</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="CleanETALabel">
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ETA: 84 Years</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QProgressBar" name="CurrentCleanTaskProgressBar">
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="StartCleaningButtonLayout">
     <item>
      <widget class="QPushButton" name="StartCleaningButton">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Start Cleaning Cycle</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
